### PR TITLE
Site redesign in cb-bootstrap

### LINKED ISSUE #226: Overlapping of text in the navbar due to scrolling.
Screenshot (8)

### LINKED ISSUE #235: Nav bar goes to bottom of the page after resizing the page
issue_3

Navigation bar goes to **bottom** of the page after **resizing**  the page **2-3 times** and doing **hover** on the **"claim"** and **"leaderboard"** option and then again resizing the window to normal resolution 

Then it **resets** to normal after **refreshing** the page.

### LINKED ISSUE #240: The tabs are not aligned properly
Selection_003
The Logout and the Welcome links are not aligned properly in Y direction. They should be realigned for better UI

### LINKED ISSUE #287: use cb bootstrap theme
- semantic ui isn't well maintained anymore (fomantic exists, but nevertheless)
- we have our own branded bootstrap edition - https://bootstrap.codingblocks.in

we will move the entire frontend (i.e. hbs pages) to cb's bootstrap and remove all semantic/fomantic stuff

### LINKED ISSUE #309: Pagination out of screen
Capture

### LINKED ISSUE #326: Improve pagination of Leaderboard.
Like that of issue #309  .
Leader board can also go out of page.

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fixes #287 
Fixes #309 
Fixes #326
Voids, Fixes #240 
Voids, Fixes #235 
Voids, Fixes #226, #200, #201
I am aware of the one pr per issue policy, but since this is a complete overhaul, some fixes had to be implemented. 

Screenshots
---
screencapture-localhost-3232-claims-add-2020-05-20-12_46_44
screencapture-localhost-3232-claims-20105-2020-05-20-12_44_26
screencapture-localhost-3232-claims-10106-2020-05-20-12_43_49
screencapture-localhost-3232-2020-05-20-12_43_02
screencapture-localhost-3232-2020-05-20-12_41_19
screencapture-localhost-3232-asdasd-2020-05-20-12_40_11
edit claim
screencapture-localhost-3232-claims-add-2020-05-20-12_38_41
screencapture-localhost-3232-stats-2020-05-20-12_38_07
screencapture-localhost-3232-claims-view-2020-05-20-12_37_58
pagination
screencapture-localhost-3232-claims-view-2020-05-20-12_37_06
screencapture-localhost-3232-claims-view-2020-05-20-12_36_57
screencapture-localhost-3232-claims-view-2020-05-20-12_36_50
screencapture-localhost-3232-leaderboard-2020-2020-05-20-12_36_35
screencapture-localhost-3232-2020-05-20-12_36_27

---

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@hereisnaman @championswimmer

**Comment by YashKumarVerma** [comment]
Requesting someone to carry it forward.

**Comment by championswimmer** [comment]
apologies for the heated discussion on the other issue. but that's in its place, and this is in its own place. 

I'll request you not to abandon other constructive work you're doing because of that particular case. upto you :)

File: .gitignore [hunk 1]
```diff
@@ -53,7 +53,7 @@ audit/*.json
 /.vscode/
 
 # Semantic UI
-/semantic/*
-!/semantic/src/site
-!/semantic/src/theme.config
+# /semantic/*
```
Comment (championswimmer) [inline .gitignore:56]: just delete the lines i'll recommend do not only comment

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
-[submodule "semantic/src/site"]
-	path = semantic/src/site
-	url = https://github.com/coding-blocks/semantic-site-config.git
+# [submodule "semantic/src/site"]
+# 	path = semantic/src/site
```
Comment (championswimmer) [inline .gitmodules:2]: blow away this submodule ! haha 
do not comment, delete the lines totally

File: secrets-sample.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
-  "secret" : "someMuchSecretFFS",
-  "clientId" : "ClientID",
-  "clientSecret" : "Client Secret",
-  "callbackURL" :  "https://boss.codingblocks.com/login/callback",
-  "BOSS_DB_NAME": "dbname",
-  "BOSS_DB_USER": "username",
-  "BOSS_DB_PASS": "pass",
+  "secret": "boss_claims",
```
Comment (championswimmer) [inline secrets-sample.json:2]: do not change these please :(

**Comment by championswimmer** [comment]
hey @YashKumarVerma are you going to be updating it ? this is kinda important for many other held PRs

File: .gitignore (hunk 1 above)
Comment (YashKumarVerma) [reply .gitignore]: resolved via new commit

File: .gitmodules (hunk 2 above)
Comment (YashKumarVerma) [reply .gitmodules]: resolved via new commit

**Pull request merged**